app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -60,6 +82,31 @@ export default function NoteTable({
                 </div>
               </td>
               <td>{note.description}</td>
+              <td className="text-center">
+                <Badge color={GetBadgeColor(note.tag)}>{note.tag}</Badge>
+              </td>
+              <td className="text-center">{note.createdDate}</td>
+              <td className="text-center">
+                {note.isDueDate ? note.dueDate : "--"}
+              </td>
+              <td>
+                <Avatar
+                  contact={{ name: note.contact }}
+                  bgClassName="bg-purple-300"
+                  className="m-auto"
+                />
+              </td>
+              <td>
+                <div className="flex flex-row">
+                  <Button style="icon" icon="ri-edit-line" className="mx-2" />
+                  <Button
+                    style="icon"
+                    icon="ri-delete-bin-5-line"
+                    className="mx-2"
```

Comment:
Rather than providing `mx-2` to individual blocks here. Please use `space-x-2`. 

Please refer [space between](https://tailwindcss.com/docs/space) from tailwindCSS for more details.